chart/values.yaml

```diff
@@ -705,6 +705,13 @@ obs:
         # NodePort associated with https port
         https: 90010
 
+storageClass:
+  enabled: true
+  parameters:
+    nameSuffix: single-replica
```

Comment:
this probably leave it on StorageClass as it's not a parameter?

Reply:
Makes sense, fixed.